src/textual/_ansi_sequences.py

```diff
@@ -1,9 +1,14 @@
 from typing import Dict, Tuple
 
+from ._terminal_modes import (
+    get__mode_report_sequence,
+    Mode,
+    ModeReportParameter,
+)
 from .keys import Keys
 
 # Mapping of vt100 escape codes to Keys.
-ANSI_SEQUENCES: Dict[str, Tuple[Keys, ...]] = {
+ANSI_SEQUENCES_KEYS: Dict[str, Tuple[Keys, ...]] = {
```

Comment:
Apart form keyboards' keys there are other ANSI sequences we might be interested, like the brand new "modes"-related ones introduced in this PR.
That's why I opted for a slightly more specific renaming of this constant